jujugui/static/gui/src/app/jujulib/test-charmstore.js

```diff
@@ -544,14 +554,27 @@ describe('jujulib charmstore', function() {
 
     it('calls the correct path', function() {
       charmstore.getEntity('cs:foobar', sinon.stub());
-      var path = charmstore.bakery.sendGetRequest.lastCall.args[0];
-      assert.equal(
-        path, 'local/v5/foobar/meta/any?include=bundle-metadata' +
+      const path = charmstore.bakery.sendGetRequest.lastCall.args[0];
+      const expectedPath = (
+        'local/v5/foobar/meta/any' +
+        '?include=bundle-metadata' +
         '&include=bundle-machine-count' +
-        '&include=charm-metadata&include=charm-config&include=common-info' +
-        '&include=id-revision&include=revision-info&include=manifest' +
-        '&include=stats&include=extra-info&include=tags&include=charm-metrics' +
-        '&include=owner&include=resources&include=supported-series');
+        '&include=charm-config' +
+        '&include=charm-metadata' +
+        '&include=charm-metrics' +
+        '&include=common-info' +
+        '&include=extra-info' +
+        '&include=id-revision' +
+        '&include=manifest' +
+        '&include=owner' +
+        '&include=published' +
+        '&include=resources' +
+        '&include=revision-info' +
+        '&include=stats' +
+        '&include=supported-series' +
+        '&include=tags'
```

Comment:
+1 for separate lines. Can we maybe change it to a ` string to get rid of the +'s?